packages/app/src/hooks/useSyncCurrentStateIntoOpenedProjects.ts

```diff
@@ -76,26 +77,26 @@ export function useSyncCurrentStateIntoOpenedProjects() {
           },
         },
         openedGraph: currentGraph.metadata!.id!,
-      });
+      } satisfies OpenedProjectInfo);
     }
   }, [currentGraph, currentProject, prevProjectState.project.metadata.id]);
 
   // Sync current graph into opened projects when user switches projects.
   useEffect(() => {
-    if (prevProjectState.project != null && prevProjectState.project.metadata.id !== currentProject.metadata.id) {
+    if (prevProjectState.project != null && prevProjectState.project.metadata.id !== currentProject.metadata.id && openedProjects[prevProjectState.project.metadata.id]) {
```

Comment:
this is the FIX here, don't add "prevProjectState" if it's no longer in "openedProjects"

Reply:
Thanks!